src/SprykerSdk/Zed/SprykGui/Communication/Form/SprykDetailsForm.php

```diff
@@ -110,10 +110,8 @@ protected function addArgumentsToForm(FormBuilderInterface $builder, array $argu
                 if (isset($argumentDefinition['isOptional'])) {
                     $typeOptions['required'] = false;
                 }
-                if (isset($argumentDefinition['isMultiple'])) {
```

Comment:
Why this was removed?

Reply:
Good question..

Reply:
Because it throws an exception, `TextType`, `TextareaType`  and `$formTypeName` all don't accept `multiple` as type option.

Only `ChoiceType` have, but this is another bug because `$formTypeName` is never evaluated to `ChoiceType` at this line.

Reply:
How about to add a test case and fix the real issue instead of removing a feature?